pysat/_instrument.py

```diff
@@ -407,30 +399,15 @@ def __getitem__(self, key):
                 except (KeyError, TypeError):
                     # TypeError for single integer
                     # KeyError for list, array, slice of integers
-                    try:
-                        # Assume key[0] is integer (including list or slice)
-                        return self.data.loc[self.data.index[key[0]], key[1]]
-                    except ValueError as err:
-                        estring = ' '.join((str(err), "\n",
-                                            "Unable to sort out data.",
-                                            "Instrument has data : ",
-                                            str(not self.empty), "\n",
-                                            "Requested key : ", str(key)))
-                        raise ValueError(estring)
+                    # Assume key[0] is integer (including list or slice)
```

Comment:
I cannot get these error messages to trigger.  The only ways I can break this involve a TypeError from pandas, not a ValueError.  We could change and test for this, but I don't think the appended message is super useful.